api/graphql/applications/application_types.py

```diff
@@ -13,7 +13,6 @@
 from api.graphql.reservation_units.reservation_unit_types import ReservationUnitType
 from api.graphql.translate_fields import get_all_translatable_fields
 from applications.models import (
-    DAY_CHOICES,
```

Comment:
Good point, it might be un-needed. Personally I'd like to have non-translated as well defined here. What we really could do is refactor to have only one DAYS since we have it elsewhere also reproduced.
So shortly I'll merge this as is and hopefully when doing enum refactoring to graphql api we can sort this one out too.